cli/src/rdwatch_cli/api/image.py

```diff
@@ -16,6 +16,39 @@ class WebpTile:
     image: Image.Image
 
 
+@dataclass
+class ObsImage:
```

Comment:
imo `ObsImage` is too unclear for someone not familiar with the naming terminology for our internal data models.
```suggestion
class ObservationImage:
```